### PR TITLE
[DeploymentScripts] Add tail query string

### PR DESCRIPTION
**Pull request by filizt**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Review by ryansbenson — COMMENTED** [review]
Approved from ARMs side

**Review by majastrz — APPROVED** [review]

**Review by raych1 — APPROVED** [review]

File: specification/resources/resource-manager/Microsoft.Resources/preview/2019-10-01-preview/examples/DeploymentScripts_GetLogsDefaultWithTail.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "parameters": {
+    "api-version": "2019-10-01-preview",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "script-rg",
+    "scriptName": "MyDeploymentScript",
+    "tail": "5"
```
Comment (raych1) [inline specification/resources/resource-manager/Microsoft.Resources/preview/2019-10-01-preview/examples/DeploymentScripts_GetLogsDefaultWithTail.json]: Type mismatch with the definition in swagger?

**Review by raych1 — CHANGES_REQUESTED** [review]
Please fix the type of 'tail' in example.

File: specification/resources/resource-manager/Microsoft.Resources/preview/2019-10-01-preview/examples/DeploymentScripts_GetLogsDefaultWithTail.json (hunk 1 above)
Comment (filizt) [reply specification/resources/resource-manager/Microsoft.Resources/preview/2019-10-01-preview/examples/DeploymentScripts_GetLogsDefaultWithTail.json]: @raych1 Fixed.

**Comment by raych1** [comment]
@NullMDR , can you please have a look at the SDK Java failures?

**Comment by NullMDR** [comment]
@raych1 It's github's failure. You can just ignore this.

**Review by raych1 — APPROVED** [review]

**Pull request merged**